### DISCUSSION TITLE
Is there a ekuiper image based on Debian with a Python environment available?

### DISCUSSION THREAD

**Discussion by silenceli** [lf-edge/ekuiper#2372] [Q&A]
The images provided in the current documentation are all based on Alpine（without python env）, but I want to create an AI function using the Python portable plugin. Therefore, I feel that a Debian environment would be much more convenient.

**Comment by ngjaying** [comment]
lfedge/ekuiper:<tag>-slim-python, Check https://hub.docker.com/r/lfedge/ekuiper image variants.
